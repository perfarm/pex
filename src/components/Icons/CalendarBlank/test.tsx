import React from 'react';

import { render } from '@testing-library/react';

import { CalendarBlank } from '.';

describe('CalendarBlank', () => {
  test('should match snapshot', () => {
    const { container } = render(<CalendarBlank />);

    expect(container).toMatchSnapshot();
  });
});
