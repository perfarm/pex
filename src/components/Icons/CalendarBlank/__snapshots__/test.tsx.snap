// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CalendarBlank should match snapshot 1`] = `
<div>
  <svg
    class="PJLV"
    fill="none"
    height="18"
    viewBox="0 0 24 24"
    width="18"
    xmlns="http://www.w3.org/2000/svg"
  >
    <path
      class="PJLV"
      d="M3.75 8.25H20.25M16.5 2.25V5.25M7.5 2.25V5.25M4.5 3.75H19.5C19.9142 3.75 20.25 4.08579 20.25 4.5V19.5C20.25 19.9142 19.9142 20.25 19.5 20.25H4.5C4.08579 20.25 3.75 19.9142 3.75 19.5V4.5C3.75 4.08579 4.08579 3.75 4.5 3.75Z"
      stroke="#898989"
      stroke-linecap="round"
      stroke-linejoin="round"
      stroke-width="1.5"
    />
  </svg>
</div>
`;
